mobs/nakki_v2/ai_states/NakkiMovementState.cs

```diff
@@ -0,0 +1,107 @@
+using Godot;
+using Godot.Collections;
+using System.Linq;
+
+using Jakojaannos.WisperingWoods.Characters.Player;
+using Jakojaannos.WisperingWoods.Util;
+using Jakojaannos.WisperingWoods.Util.Editor;
+
+namespace Jakojaannos.WisperingWoods;
+
+[Tool]
+public partial class NakkiMovementState : NakkiAiState {
+	public override string[] _GetConfigurationWarnings() {
+		var warnings = base._GetConfigurationWarnings() ?? System.Array.Empty<string>();
+
+		if (_pickOneOfTheseStatesWhenDoneMoving.Count == 0) {
+			warnings = warnings.Append("Add one or more states to 'pick one of these when done'-list").ToArray();
+		}
+
+		return warnings
+			.Union(this.CheckCommonConfigurationWarnings())
+			.ToArray();
+	}
+
+	[Export] private Array<NakkiAiState> _pickOneOfTheseStatesWhenDoneMoving = [];
+
+	[Export] private float _moveTime = 3.0f;
+	[Export] private float _moveTimeVariation = 0.3f;
+	[Export] private float _moveToPlayerChance = 0.2f;
+
+	[Export]
+	[MustSetInEditor]
+	public NakkiStalkState StalkState {
+		get => this.GetNotNullExportPropertyWithNullableBackingField(_stalkState);
+		set => this.SetExportProperty(ref _stalkState, value, notifyPropertyListChanged: true);
```

Comment:
Is the `notifyPropertyListChanged: true` actually needed for these? I think its only needed for cases where there are some `_ValidateProperty` or `_GetPropertyList` magic going on, or what am I forgetting? :thinking: